facebookresearch/ParlAI
# Added easy Colab notebook .ipynb and Colab link in Readme.md

**Patch description**
This adds a link to a Colab notebook for easy installation and use of the interactive version of this repo, as described on the Parl.ai website.

**Testing steps**
1.  Click the "Colab" shield at the top of Readme.md.
2.  Once the Colab notebook opens, switch runtime to GPU
3.  Execute all cells.  The 3B model should now run interactively.
